src/lib/containers/GenericCard.tsx

```diff
@@ -255,21 +257,32 @@ export default class GenericCard extends React.Component<
     } else if (!titleLink) {
       target = '_blank'
     } else if (titleLink) {
+      href = this.getCardLinkHref(titleLink, data, schema) ?? ''
```

Comment:
refactored this code into a shared getCardLinkHref() function, reused for the new cta button (when link is defined).